test/plugins/pylint_test.py

```diff
@@ -94,7 +95,10 @@ def test_that_we_can_test_pylint():
         # TODO is checked by an IRawChecker. #=C
         """
     msg_ids = "unused-import,redefined-builtin,anomalous-backslash-in-string,fixme"
```

Comment:
Add __invalid-escape__ to __msg_ids__ ?

Reply:
`invalid-escape` isn't a pylint message.  `anomalous-backslash-in-string` is.  The problem you reported was that Python is warning about `\c` while pytest is also warning about it.  The pytest violation is being handed in the test. We needed to suppress the Python warning.